docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -55,7 +55,7 @@ Create `alertmanager-config.yaml` file if necessary. Visit the [alertmanager
 configuration](https://prometheus.io/docs/alerting/configuration/#configuration-file) for more
 information.
 
-**Note**: Make sure it is indented to two spaces.
+**Note**: Make sure the whole file is indented two spaces. That is, there are two spaces before the top level block.
```

Comment:
Hm, that doesn't make any sense to me. Why can't we ensure the indentation when we render the library?

Reply:
Yep, that's a good point. A better UX is handling that for the user.

Reply:
We can do this:

```diff
diff --git a/pkg/components/prometheus-operator/template.go b/pkg/components/prometheus-operator/template.go
index 02d1b849..d630c38d 100644
--- a/pkg/components/prometheus-operator/template.go
+++ b/pkg/components/prometheus-operator/template.go
@@ -16,7 +16,7 @@ package prometheus
 
 const chartValuesTmpl = `
 alertmanager:
-{{.AlertManagerConfig}}
+  {{.AlertManagerConfig}}
   alertmanagerSpec:
     retention: {{.AlertManagerRetention}}
     externalUrl: {{.AlertManagerExternalURL}}
```

What do you think, @surajssd?

Reply:
You then need to indent a whole block. E.g. https://github.com/flexkube/libflexkube/blob/master/internal/util/util.go#L55.

Reply:
Just doing that won't put the whole config two spaces forward. We need to manually (or programmatically indent it like that).

Or other fool proof solution is to marshal the yaml into the same config prom operator uses and then unmarshal it into the file with whatever indentataion we want.

Reply:
Can we merge this to fix the ambiguity and create an issue to fix it?